src/NuGetGallery/App_Start/DefaultDependenciesModule.cs

```diff
@@ -442,6 +442,10 @@ protected override void Load(ContainerBuilder builder)
                 .As<IPackageVulnerabilityService>()
                 .InstancePerLifetimeScope();
 
+            builder.Register(c => new CookieExpirationService(domain: configuration.GetSiteRoot(true)))
```

Comment:
Can you use a named parameter here? It makes it a little clearer what the `true` is for:

```suggestion
            builder.Register(c => new CookieExpirationService(domain: configuration.GetSiteRoot(useHttps: true)))
```